src/modelinstance.cpp

```diff
@@ -413,6 +414,29 @@ Status ModelInstance::loadTensors(const ModelConfig& config, bool needsToApplyLa
         SPDLOG_LOGGER_ERROR(modelmanager_logger, "Error during configuration validation against model");
         return status;
     }
+    // add output names if not present in the model
```

Comment:
Remove comment and make function 
adjustForEmptyOutputNames/addOutputNamesIfNotPresent or sth similar to operate on similar level of abstraction inthis function - see loadInputTensors/applyLayoutConfiguration etc